packages/vscode-extension/src/builders/PlatformBuildCache.ts

```diff
@@ -53,21 +54,34 @@ export class PlatformBuildCache {
   /**
    * Passed fingerprint should be calculated at the time build is started.
    */
-  public async storeBuild(buildFingerprint: string, build: BuildResult) {
+  public async storeCache(buildFingerprint: string, build: BuildResult) {
     const appPath = await getAppHash(getAppPath(build));
-    await extensionContext.workspaceState.update(this.cacheKey, {
+
+    const cache = JSON.stringify({
       fingerprint: buildFingerprint,
       buildHash: appPath,
       buildResult: build,
     });
+
+    setAppCache(this.cacheKey, cache);
```

Comment:
In contrast, this method actually stores stuff, so why is it called "set" – this is a frequently used prefix for setting class members and using it for method that writes to disk is very confusing.